basics.js

```diff
@@ -1,24 +1,41 @@
 /* Create a `myName` variable and assign it a String value */
+var myName = "Dev League";
 
 /* Create a `person` variable and give it 2 properties,
  * `name`, assign it the same name as before,
  * as well as an `age` (number);
  */
-
+var person = {
+	name: myName, 
+	age:33
+};
 /* Create a function called `greet`,
- * it should take a 1 parameter, `name`
+ * it should take a 1 parameter, `name`r
  * and it should print "Hello, my name is {name}"
  */
 
+function greet (name) {
+	name = person.name;
```

Comment:
No need to assign name to `person.name`, that would mean it could only greet 1 person